### ISSUE TITLE
Setting to hide notifications on camera

### ISSUE THREAD

**Issue by linki9s7** [Yellow-Dog-Man/Resonite-Issues#1991]
### Is your feature request related to a problem? Please describe.

With this feature in place, when recording a video, you will no longer see a notification on your camera that someone is online, for example.

### Describe the solution you'd like

Add a toggle to show/hide the camera in the notifications section of settings.

### Describe alternatives you've considered

An alternative is to temporarily hide notifications. (like focus mode)

### Requesters

Username / Discord ↓
linki9s7 / linki9s7

**Comment by Frooxius** [comment]
I'm not sure why would this need to be its own setting? They only show if you specifically enable rendering private UI. And you can just set yourself as Busy to stop showing the settings.

What are the usecases?

**Comment by shadowpanther** [comment]
> What are the usecases?

I'd imagine streaming or recording the private UI for educational/tutorial purposes, so maybe add an option to disable the notifications when the streaming (LIVE) mode is on?

On the other hand, as you said, just switching to Busy gets rid of the popups.

**Comment by TisFoolish** [comment]
> I'd imagine streaming or recording the private UI for educational/tutorial purposes, so maybe add an option to disable the notifications when the streaming (LIVE) mode is on?
> 
> On the other hand, as you said, just switching to Busy gets rid of the popups.

To me that sounds like would be better solved with a sort of "streaming" status, something that hides certain things in the private UI from the stream camera while also letting your contacts know that you're streaming.

**Comment by shadowpanther** [comment]
> To me that sounds like would be better solved with a sort of "streaming" status

Yes, the LIVE status, as I said. I don't think it actually changes much currently, but settings could be added to hide notifications, go to Busy, show that you are streaming to Contacts, etc.

**Comment by TisFoolish** [comment]
That's a badge, not a status. I meant something along the lines of the Online, Busy, Away, and Invisible statuses

**Comment by Frooxius** [comment]
Is there a reason to not just set yourself as "Busy" though?

Because right now, this setting would be useful if you need to have Private UI rendering on, while at the same time you don't want to have Busy status.

**Comment by JackTheFoxOtter** [comment]
Personally I'd say busy is what you want in this situation, I just don't think many people know that suppresses notifications.

**Comment by epicEaston197** [comment]
> > What are the usecases?
> 
> I'd imagine streaming or recording the private UI for educational/tutorial purposes, so maybe add an option to disable the notifications when the streaming (LIVE) mode is on?
> 
> On the other hand, as you said, just switching to Busy gets rid of the popups.

This setting already exists it's called the user online notification you could just turn all the notifications off if you don't want to be in busy mode

**Comment by shiftyscales** [comment]
Marking as blocked by #1918, thanks. :)
